examples/label_studio_annotation/README.md

```diff
@@ -0,0 +1,318 @@
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetObject",
+                "s3:PutObject",
+                "s3:DeleteObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::<YOUR_BUCKET_NAME>",
+                "arn:aws:s3:::<YOUR_BUCKET_NAME>/*"
+            ]
+        }
+    ]
+}
+```
+
+Create a policy using [the `create-policy` CLI command](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create-cli.html) for IAM to manage the Label Studio access to the S3 bucket. This assumes you are running the command from the same directory in which you saved the policy file.
+
+```shell
+# this token duration is the maximum possible, but you will have to check if your bucket allows this
+aws iam create-policy --policy-name <YOUR_POLICY_NAME> --policy-document file://<YOUR_POLICY_FILE_NAME> --duration_seconds=43200
+```
+
+Set up cross-origin resource sharing (CORS) access to your bucket, using a policy that allows `GET` access from the same host name as your Label Studio deployment. You can use the CORS file we have pre-populated for you inside the examples repo as follows:
+
+```shell
+cd cloud_config/aws
+
+aws s3api put-bucket-cors --bucket $S3_BUCKET_NAME --cors-configuration file://cors.json
+
+cd ../..
+```
+
+Now you can get to the fun part of setting up Label Studio and working with ZenML:
+
+```shell
+zenml stack copy default <YOUR_AWS_ZENML_STACK_NAME>
+
+zenml stack set <YOUR_AWS_ZENML_STACK_NAME>
+
+zenml secrets-manager register <YOUR_SECRETS_MANAGER> --flavor=aws --region_name=<AWS_REGION_NAME>
+
+zenml stack update <YOUR_AWS_ZENML_STACK_NAME> -x <YOUR_SECRETS_MANAGER>
+
+# copy down the credentials output from this command
+aws sts get-session-token --profile=<YOUR_AWS_CONFIG_PROFILE_NAME>
+```
+
+Use the credentials output from the command to get your `<LABEL_STUDIO_ACCESS_KEY_ID>`, `<LABEL_STUDIO_SECRET_ACCESS_KEY>`, and `<LABEL_STUDIO_AWS_SESSION_TOKEN>` which we will use to register a secret.
+
+```shell
+# this secret must be named 'aws_label_studio'
+zenml secret register aws_label_studio --schema=aws_label_studio --aws_access_key_id="<LABEL_STUDIO_ACCESS_KEY_ID>" --aws_secret_access_key="<LABEL_STUDIO_SECRET_ACCESS_KEY>" --aws_session_token="<LABEL_STUDIO_AWS_SESSION_TOKEN>"
+
+# use your standard access key id and secret access key from ~/.aws/credentials here
+zenml secret register <YOUR_AWS_SECRET_NAME> --schema=aws_label_studio--aws_access_key_id="<YOUR_ACCESS_KEY_ID>" --aws_secret_access_key="<YOUR_SECRET_ACCESS_KEY>"
+
+zenml artifact-store register <YOUR_CLOUD_ARTIFACT_STORE> --flavor=s3 --path=s3://<YOUR_S3_BUCKET_NAME> --authentication_secret="<YOUR_AWS_SECRET_NAME>"
+
+zenml stack update <YOUR_AWS_ZENML_STACK_NAME> -a <YOUR_CLOUD_ARTIFACT_STORE>
+```
+
+You will next need to obtain your Label Studio API key. This will give you access to the web annotation interface.
```

Comment:
Can we move this up ahead of the split into the three cloud providers to reduce the duplication?